### DISCUSSION TITLE
Need Help with iOS 14 DNS

### DISCUSSION THREAD

**Discussion by alphanull** [paulmillr/encrypted-dns#59] [General]
Hello, this request might be slightly offtopic, but I am pulling out my hair with iOS 14 encrypted DNS and maybe someone her ecan help me. So the question is if it is possible to extend your profiles with a second, local and most importantly _NON ENCRYPTED_ DNS server? Problem is that iOS 14 seems to totally ignore such unencrypted servers if there an encrypted one (here: Cloudflare) is available. So the idea is to construct a mobileconfig which instructs iOS to use this second DNS as well, but only for certain domains. Is this possible at all?

**Comment by paulmillr** [comment]
Not sure about this, maybe Apple folks can answer

**Comment by alphanull** [comment]
So far, even no answer from Apple itself, or anyone else, so this post was made out of sheer desparation ...

**Comment by ghost** [comment]
@alphanull This is actually not possible on IOS 14/15 & Mac OS 11/12 (as far a I know).
Because profile by essence is normally for professional people who don't really want a way to do fallback on unencrypted since if they push an encrypted (forced) by profile, they normally have control over the network to let this kind of data be functional.

**Comment by ghost** [comment]
@paulmillr Since it's not a bug in .mobileconfig file I ask this issue be closed.
We cannot fix it only apple can so I vote to close it but it's up to you.
